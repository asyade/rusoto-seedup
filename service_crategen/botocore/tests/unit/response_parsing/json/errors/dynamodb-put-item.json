{"__type":"com.amazonaws.dynamodb.v20111205#ProvisionedThroughputExceededException",
    "message":"The level of configured provisioned throughput for the table was exceeded.  Consider increasing your provisioning level with the UpdateTable API"}
