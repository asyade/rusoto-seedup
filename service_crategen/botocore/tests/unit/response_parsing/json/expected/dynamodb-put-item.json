{
    "ResponseMetadata": {},
    "Error": {
        "Message": "The level of configured provisioned throughput for the table was exceeded.  Consider increasing your provisioning level with the UpdateTable API",
        "Code": "ProvisionedThroughputExceededException"
    }
}
